### PR TITLE
doc: add missing networkRBAC plugin

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

File: types/plugins/plugins_test.go [hunk 1]
```diff
@@ -59,7 +59,13 @@ func TestCheckPluginAttributes(t *testing.T) {
 			return nil
 		}
 
-		plugin := snakeToCamel(filepath.Base(path)[:len(filepath.Base(path))-3])
+		var plugin string
+		filename := filepath.Base(path)[:len(filepath.Base(path))-3]
+		if filename == "network_rbac" {
+			plugin = "networkRBAC"
```
[the remark this answers is not in the document]
Comment (spacewander) [reply types/plugins/plugins_test.go:65]: I hope snakeToCamel can only focus on the work that its name indicates.

[the remark this answers is not in the document]
Comment (zll600) [reply types/plugins/plugins_test.go:65]: IMO, `network_rbac => networkRBAC` is a special case for `snakeToCamel`. Placing this case into `snakeToCamel` will make the logic clearer. 

This is just my preference, both ways are OK.

[the remark this answers is not in the document]
Comment (spacewander) [reply types/plugins/plugins_test.go:65]: If there are some places that share the same name conversion, I would create a new function `filenameToPluginName` which contains `network_rbac => networkRBAC`.

[the remark this answers is not in the document]
Comment (zll600) [reply types/plugins/plugins_test.go:65]: > If there are some places that share the same name conversion, I would create a new function filenameToPluginName which contains network_rbac => networkRBAC.

SGTM

**Pull request merged**